app/src/main/java/org/fossasia/openevent/app/common/data/network/EventService.java

```diff
@@ -18,6 +19,9 @@
 
 public interface EventService {
 
+    @POST("users")
+    Observable<User> signUp(@Body SignUp signUp);
```

Comment:
@iamareebjamal the error showing is 
Content-Type header must be application/vnd.api+json